packages/manager/apps/hpc-vmware-managed-vcd/src/pages/dashboard/datacentre/compute/DatacentreCompute.page.tsx

```diff
@@ -84,27 +84,29 @@ export default function ComputeListingPage() {
   ];
 
   return (
-    <DatagridContainer
-      title={VHOSTS_TITLE}
-      containerId={`compute-${id}-${vdcId}`}
-      columns={columns}
-      route={{
-        api: getVcdDatacentreComputeRoute(id, vdcId),
-        onboarding: urls.datacentreComputeOrder
-          .replace(subRoutes.dashboard, id)
-          .replace(subRoutes.vdcId, vdcId),
-      }}
-      isEmbedded
-      orderButton={
-        <OsdsButton
-          size={ODS_BUTTON_SIZE.sm}
-          variant={ODS_BUTTON_VARIANT.stroked}
-          color={ODS_THEME_COLOR_INTENT.primary}
-          onClick={() => navigate(subRoutes.order)}
-        >
-          {t('managed_vcd_vdc_compute_order_cta')}
-        </OsdsButton>
-      }
-    />
+    <>
+      <DatagridContainer
+        title={VHOSTS_TITLE}
+        containerId={`compute-${id}-${vdcId}`}
+        columns={columns}
+        route={{
+          api: getVcdDatacentreComputeRoute(id, vdcId),
+          onboarding: urls.datacentreComputeOrder
+            .replace(subRoutes.dashboard, id)
+            .replace(subRoutes.vdcId, vdcId),
+        }}
+        isEmbedded
+        orderButton={
+          <OsdsButton
+            size={ODS_BUTTON_SIZE.sm}
+            variant={ODS_BUTTON_VARIANT.stroked}
+            color={ODS_THEME_COLOR_INTENT.primary}
+            onClick={() => navigate(subRoutes.order)}
+          >
+            {t('managed_vcd_vdc_compute_order_cta')}
+          </OsdsButton>
+        }
+      />
+    </>
```

Comment:
Is the fragment wrapping `<DatagridContainer />` needed for something ?

Reply:
No I removed it 👍🏾